src/Education/XamlGenerator/IRuleHelpXamlBuilder.cs

```diff
@@ -297,10 +296,8 @@ private void ProcessElement()
                     writer.WriteStartElement("TableCell");
                     PushOutputElementInfo("td", false);
 
-                    if (tableAlternateRow)
-                    {
-                        ApplyStyleToStartElement(StyleResourceNames.TableBodyCellAlternateRow);
-                    }
+                    var cellStyle = tableAlternateRow ? StyleResourceNames.TableBodyCellAlternateRow : StyleResourceNames.TableBodyCell;
```

Comment:
I also added a style for "normal" table cells. That means we can define one style for the odd-numbered rows, and another for the even-numbered rows.

Reply:
Ah very nice!